conf/php/php-fpm.conf

```diff
@@ -196,7 +196,7 @@ pm = static
 ; forget to tweak pm.* to fit your needs.
 ; Note: Used when pm is set to 'static', 'dynamic' or 'ondemand'
 ; Note: This value is mandatory.
-pm.max_children = 8
+pm.max_children = 3
```

Comment:
I am using this with dokku on a server with ~10 apps and the memory usage is a bit much with 8 php processes per app - epecially since I do not have 8 cores. I have moved this customization into another branch.